debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -836,8 +927,24 @@ public static final Field MASK_COLUMN(int length) {
                                                                 DatabaseHistory.STORE_ONLY_MONITORED_TABLES_DDL,
                                                                 DatabaseHistory.DDL_FILTER);
 
+    private final SnapshotLockingMode snapshotLockingMode;
+    @Deprecated
+    private final boolean snapshotMinimalLocking;
```

Comment:
Do we even need this as a field any longer? It seems both config values could be "normalized" into `snapshotLockingMode` (giving the new option precedence), allowing the consuming code just to deal with that single option.